src/lib/components/table/table-paging-data-source.spec.ts

```diff
@@ -0,0 +1,53 @@
+import { TablePagingDataSource } from './table-paging-data-source';
+import { RequestParameterInterface } from './request-parameter.interface';
+import {
+    Observable,
+    of
+} from 'rxjs';
+import {
+    MatPaginator,
+    MatPaginatorIntl
+} from '@angular/material/paginator';
+
+class TestDataSource extends TablePagingDataSource<{}>
+{
+    public request(requestParams:RequestParameterInterface):Observable<Array<{}>>
+    {
+        return of(new Array({}));
+    }
+}
```

Comment:
Ggf. in eigene Datei auslagern? Wiederholt sich

Reply:
Wiederholt sich nur bedingt, weil immer eine andere BaseClass extended wird

Reply:
Hast recht.. Werde unaufmerksam 🙈